apps/studio/src/features/editing-experience/LayoutSelection.tsx

```diff
@@ -0,0 +1,438 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable prettier/prettier */
+/* eslint-disable */
+
+// Component for layout selection
+
+import React, { useState } from 'react'
+import {
+  Box,
+  Center,
+  Grid,
+  GridItem,
+  HStack,
+  Image,
+  List,
+  ListItem,
+  Text,
+  VStack,
+  useToken,
+} from '@chakra-ui/react'
+import { Button, BxRightArrowAlt } from '@opengovsg/design-system-react'
+import Preview from '../editing-experience/components/Preview'
+
+import { Icon } from '@chakra-ui/react'
+import { BiLeftArrowAlt, BiShow } from 'react-icons/bi'
+
+interface LayoutSelectionProps {
+  pageName: String
+}
+
+const LAYOUT_DATA: {
+  layoutDisplayName: string
+  layoutSchemaName: string
+  layoutDescription: string
+  imageSource: string
+}[] = [
+  {
+    layoutDisplayName: 'Default',
+    layoutSchemaName: 'content',
+    layoutDescription: 'This is the most basic layout for your content.',
+    imageSource: '/assets/layout_card/default_layout_card.png',
+  },
+  {
+    layoutDisplayName: 'Article',
+    layoutSchemaName: 'article',
+    layoutDescription:
+      'Designed for the perfect reading experience. Use this layout for text-heavy content, such as news, press releases, and speeches',
+    imageSource: '/assets/layout_card/article_layout_card.png',
+  },
+]
+
+const placeholderSchema = {
```

Comment:
can we extract the schemas out into a different file to export these so that the code is clean